### ISSUE TITLE
Warning with new ember-cli 2.13: ignoring input sourcemap because ENOENT

### ISSUE THREAD

**Issue by vmalloc** [gcollazo/ember-cli-showdown#49]
I have ember-cli-showdown installed and recently upgraded to ember-cli 2.13. Now I get this warning when building:
```
Warning: ignoring input sourcemap for vendor/showdown.js because ENOENT: no such file or directory, open 'src/backslash/webapp/tmp/source_map_concat-input_base_path-LjkWWUdI.tmp/vendor/showdown.js.map
```
Is this a known issue?

**Comment by jasonmit** [comment]
If you can throw together an example app that demonstrates it I can help triage.  No one has reported yet, so needs investigation.

**Comment by jasonmit** [comment]
Fixed in 3.1.2

**Comment by vmalloc** [comment]
@jasonmit sorry I missed your first comment in my inbox! Glad you can find it and fix it nevertheless.
